jest.config.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  testEnvironment: 'node',
+  collectCoverage: true,
+  collectCoverageFrom: [
+    'lib/**/*.js'
+  ],
+  moduleNameMapper: {
+    '^~/(.*)$': '<rootDir>/lib/$1',
+    '^~~$': '<rootDir>',
+    '^@@$': '<rootDir>',
+    '^@/(.*)$': '<rootDir>/lib/$1'
+  },
+  transform: {
```

Comment:
No need `transform`

Reply:
OK, but `lib/module.js` uses `export default function` (ES module). 
There is a need to change it to `module.exports = function` (CommonJS), if uses `{ handler: require('../../../') }`.

which is better?

Reply:
Use `CommonJS` to be more backwards compatible with nuxt